pkg/fixtures/test/panic_on_no_return_value_test.go

```diff
@@ -0,0 +1,23 @@
+package test
```

Comment:
@LandonTClipp I was wondering if this is the right package for this test, but as you suggested to create a new one, I created a new file and placed it down next to the fixture that I just created, Let me know if there is another better package for this.

Reply:
You can just put it in the same directory as the fixture. Thanks!

Reply:
Thanks for all your code suggestions, @LandonTClipp! At this specific point, I guess it won't be possible as we have an import cycle here because of one of the imports used in the test file [here](https://github.com/vektra/mockery/pull/736/files#diff-eaff0e9a7c5657f64f7817700c73dd7ed611083fd46e32faced795b83ffa7873R8).

Reply:
Not sure I understand. Placing the test next to `pkg/fixtures/panic_err.go` makes an import cycle?

Reply:
I discovered why we had an import cycle here, the package from the **pkg/fixtures/panic_err.go**  file was named "`package test`" (like in the other files next to it), and the package in **panic_err_test.go** was also named "test", so renaming it to "`package test_test`" works. Just an example to improve this... we should rename the entire package name from other nontest files to "fixtures" and use in test files something like this "`package fixtures_test`".

Let me know if it's clear.

Reply:
Ah! Okay, that's interesting. I think I'll want to change how the fixtures are structured, but I think your solution is good for now. Thanks.